marathon_acme/clients.py

```diff
@@ -222,20 +222,20 @@ def raise_for_not_ok_status(response):
     return response
 
 
-def _sse_content_with_protocol(response, handler):
+def _sse_content_with_protocol(response, handler, timeout=None, reactor=None):
```

Comment:
I don't actually have a test that these new parameters get passed all the way through. Trying to figure that out.